app_feup/lib/controller/local_storage/app_exams_database.dart

```diff
@@ -36,6 +36,12 @@ class AppExamsDatabase extends AppDatabase {
           maps[i]['month'],
           maps[i]['year']);
     });
+    
+    list.sort((exam1, exam2) {
+      return exam1.date.compareTo(exam2.date);
+    });
+
+    return list;
```

Comment:
I think we can remove this. Whenever you'll open up the app, exams will be updated nonetheless.